pack/ramp-enterprise.yml

```diff
@@ -53,8 +53,6 @@ exclude_commands:
     - _FT.CURSOR
     - _FT.SPELLCHECK
     - _FT.TAGVALS
-    - _FT.DEL
-    - _FT.SAFEADD
```

Comment:
Turns out we have a legacy `_FT.DEL` command that we (probably don't) need to still support on Enterprise.